lib/Doctrine/Table.php

```diff
@@ -1704,7 +1704,7 @@ public function findByDql($dql, $params = array(), $hydrationMode = null)
      * Find records basing on a field.
      *
      * @param string $column            field for the WHERE clause
-     * @param string $value             prepared statement parameter
+     * @param string|array $value       prepared statement parameter
```

Comment:
These two are not related to 8.1 as well, but it's nice in editors with code assist to know the correct types.